doc/nrf/gs_programming.rst

```diff
@@ -34,11 +34,15 @@ Complete the following steps to build |NCS| projects with SES after :ref:`instal
 
 #. To import a project into SES, you must specify the following information:
 
-   * **CMakeLists.txt** - the location of the :file:`CMakeLists.txt` project file of the sample that you want to work with
-   *  **Board Directory** - the location of the board description of the board for which to build the project
-   *  **Board Name** - the board name (select from the list that is populated based on the board directory)
-   * **Build Directory** - the folder in which to run the build (automatically filled based on the board name, but you can specify a different directory)
-   * **Delete Existing CMakeCache.txt** - select this option to ensure that you are not building with an outdated build cache
+   * **Projects** - Select the sample that you want to work with
+   *  **Board Name** - Select the board that you want to work with
+   *  **Build Directory** - the folder in which to run the build (automatically filled based on the board name, but you can specify a different directory)
+   * **Clean Build Directory** - select this option to ensure that you are not building with an outdated build cache
+   * **Extended Settings** - Provides a field for specifying additional CMake arguments
+
+   .. note::
+	  To select a custom project, click click :guilabel:`...` next to projects list for opening the browse dialog and manual select a project.
```

Comment:
The double click is probably not intended. I also think that the second part of the sentence is not needed. 'To select a custom project, click :guilabel:`...` next to projects list.'

Reply:
Done.